src/features/user/UserProfile/components/MyTrees/MyTrees.tsx

```diff
@@ -0,0 +1,125 @@
+  React.useEffect(() => {
+    async function loadFunction() {
+      getRequestWithoutRedirecting(`/app/profiles/${profile.id}/contributions`)
+        .then((result: any) => {
+          setContributions(result);
+        })
+        .catch((e: any) => {
+          console.log('error occured :', e);
+        });
+    }
+    loadFunction();
+  }, []);
+
+  const MapProps = {
+    contributions,
+  };
+  return (
+    <>
+      {contributions ? (
+        <div className={styles.myTreesSection}>
+          <div className={styles.myTreesTitle}>{t('me:myForest')}</div>
+
+          <div className={styles.myTreesContainer}>
+            <div className={styles.treesList}>
+              {Array.isArray(contributions) && contributions.length !== 0
+                ? contributions.map((item: any) => {
+                    return (
+                      <div className={styles.tree}>
+                        <div className={styles.dateRow}>
+                          {formatDate(item.properties.plantDate)}
+                        </div>
+                        <div className={styles.treeRow}>
+                          <div className={styles.textCol}>
+                            <div className={styles.title}>
+                              {item.properties.type === 'registration'
+                                ? t('me:registered')
+                                : item.properties.project?.name}
+                            </div>
+                            <div className={styles.country}>
+                              {item.properties.country
+                                ? t(
+                                    'country:' +
+                                      item.properties.country.toLowerCase()
+                                  )
+                                : null}
+                            </div>
+                            {item.properties.type === 'gift' ? (
+                              <div className={styles.source}>
+                                {t('me:receivedTrees')}
+                              </div>
+                            ) : null}
+                            {item.properties.type === 'redeem' ? (
+                              <div className={styles.source}>
+                                {t('me:redeemedTrees')}
+                              </div>
+                            ) : null}
+                          </div>
+                          <div className={styles.numberCol}>
+                            <div className={styles.treeIcon}>
+                              <div
+                                style={
+                                  item.properties.type === 'registration'
+                                    ? { color: '#3D67B1' }
+                                    : {}
+                                }
+                                className={styles.number}
+                              >
+                                {item.properties.treeCount}
```

Comment:
Format this number once this branch and #526 is merged!

Reply:
This is just a reminder for myself at https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/549